digits/templates/home.html

```diff
@@ -30,6 +30,44 @@
     {% endif %}
     <h1>Home</h1>
 </div>
+<script>
+function deleteWorkspace(workspace) {
+    bootbox.confirm(
+            'Are you sure you want to delete this Workspace?<br><br>All related Jobs will be permanently removed.',
+            function(result) {
+                if (result)
+                    $.ajax('/workspace/delete' +'?workspace={{ workspace }}' ,
+                        {type: "DELETE"})
+                    .done(function() {
+                        window.location = '/workspaces';
+                        })
+                    .fail(function(data) { errorAlert(data); });
+            });
+    return false;
+    }
+</script>
+<br>
+        <div class="row">
+        <div class="col-sm-8">
+
+            <form action="/workspace/edit/{{ workspace }}" method="post">
+            <div id="show-workspace-name" class="col-sm-12">
+                <h2 style="display:inline;">{{ workspace }}</h2>
```

Comment:
Can we find a way to improve the look of this? A huge "Default" without any context at the top of the page is a bit odd and confusing.

Maybe it would be clearer if (1) the current workspace, (2) the link to the list of workspaces and (3) the button to delete the workspace were all next to each other? Maybe we should move more of this up into the navbar? You can take a look at what I did for the login PR for inspiration if it's helpful - #463.